src/solvers/flattening/boolbv.cpp

```diff
@@ -639,35 +639,25 @@ void boolbvt::set_to(const exprt &expr, bool value)
   return supert::set_to(expr, value);
 }
 
-void boolbvt::make_bv_expr(const typet &type, const bvt &bv, exprt &dest)
+exprt boolbvt::make_bv_expr(const typet &type, const bvt &bv)
 {
-  dest=exprt(ID_bv_literals, type);
+  exprt dest(ID_bv_literals, type);
   irept::subt &bv_sub=dest.add(ID_bv).get_sub();
-
   bv_sub.resize(bv.size());
 
   for(std::size_t i=0; i<bv.size(); i++)
     bv_sub[i].id(std::to_string(bv[i].get()));
+  return dest;
```

Comment:
Nit pick: a blank line before `return`would help readability.

Reply:
I'd say removing double blank lines before this for loop would.